counting words in file.py

```diff
@@ -13,6 +11,7 @@ def bubble_sort(list):
                 list[x-1]=list[x]
                 list[x]=temp
                 switched=True
+#Output: The list in descending order
```

Comment:
We can read the comments faster if the Output comment is before the function definition, right below the Input comment